src/RelayPricer.ts

```diff
@@ -1,33 +1,21 @@
 import BigNumber from 'bignumber.js';
+import BaseExchangeApi from './api/BaseExchangeApi';
 import CoinBase from './api/CoinBase';
+import RdocExchange from './api/RdocExchange';
 import TestExchange from './api/TestExchange';
-import ExchangeApi from './types/ExchangeApi';
 
 const INTERMEDIATE_CURRENCY = 'USD';
 
 const coinbase = new CoinBase();
 const testExchange = new TestExchange();
+const rdocExchange = new RdocExchange();
 
-type AvailableApi = {
-    api: ExchangeApi;
-    tokens: Array<string>;
-};
-
-const EXCHANGE_APIS: Array<AvailableApi> = [
-    {
-        api: coinbase,
-        tokens: ['RIF']
-    },
-    {
-        api: testExchange,
-        tokens: ['TKN']
-    }
-];
+const EXCHANGE_APIS: BaseExchangeApi[] = [coinbase, testExchange, rdocExchange];
 
 export default class RelayPricer {
-    findAvailableApi(token: string): AvailableApi {
-        const availableApi = EXCHANGE_APIS.find((x) =>
-            x.tokens.includes(token)
+    findAvailableApi(token: string): BaseExchangeApi {
+        const availableApi = EXCHANGE_APIS.find((api) =>
+            api.getTokens().includes(token)
```

Comment:
NIT: if you accept the changes to the `BaseExchangeApi` constructor, we need to change this 
```suggestion
            api.tokens().includes(token)
```

Reply:
Done.